store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorage.java

```diff
@@ -439,26 +438,18 @@ private void updateCacheWithNewGraphId(final String newGraphId, final GraphSeria
         }
     }
 
+    /* For Accumulo derived stores the table name must
+     * be updated to match the graph's new id.
+     * These stores are:
+     * uk.gov.gchq.gaffer.accumulostore.[AccumuloStore, SingleUseAccumuloStore,
+     * MiniAccumuloStore, SingleUseMiniAccumuloStore]
+     */
```

Comment:
Is it worth making this a proper javadoc and moving this comment to the actual `renameTable()` call?

Reply:
I'd considered making it proper Javadoc but as this is a private method I didn't see much reason to. Do you mean moving this to where it was before or somewhere else? This comment relates to the Federated Store so it shouldn't be in the Accumulo module.

Reply:
I'm in the camp believing that all methods even private ones should have javadocs on as they can help developers understand the code but it is bit of a debated topic, I more meant this comment isn't really generic enough to relate to the method it should be where the method is called.

Reply:
I've moved the comment to where the method is called. This fits in with other comments in that area - I agree this is a better place for it.